packages/api-request-builder/src/create-service.js

```diff
@@ -74,6 +76,13 @@ export default function createService(
           ...queryId,
         }
 
+      if (feature === defaultFeatures.queryLocation)
+        return {
+          ...acc,
+          ...queryLocation,
+          ...queryExpand,
```

Comment:
Why do you also add this here?

Reply:
you can use expand parameter while querying `shippingMethods` by location. At least this is how I understood that this code works.

Reply:
@junajan Yes, and that's why in the `default-services.js` file, `features.queryExpand` is listed as one of the features. This should not be added here

Reply:
Removed here 13254e2 .. Hope now it is ok.